roles/edpm_nova/tasks/configure.yml

```diff
@@ -118,3 +119,24 @@
     mode: '0600'
     owner: nova
     group: nova
+
+- name: Get fact for tls_certs_enabled
+  ansible.builtin.set_fact:
+    tls_certs_enabled: "{{ lookup('env', 'tls_certs_enabled') | default(false, true) }}"
```

Comment:
I dont think this is correct we should not be using environmental variable in the AEE container to enable this

this should be enabled via ansible_vars which will be set in the inventory.

Reply:
done